docs/index.md

```diff
@@ -44,7 +44,7 @@ Workspaces can be enhanced with one or more building blocks called **workspace s
 
 Multiple workspaces can be created within a single Trusted Research Environment to enable the required separation for your projects.
 
-Each workspace has [workspace users](azure-tre-overview/user-roles.md): one **workspace owner**, and one or more **workspace researchers** that can access the data and workspace services in the workspace. The workspace owner is also considered a workspace researcher.
+Each workspace has [workspace users](azure-tre-overview/user-roles.md): a **workspace owner** (normally only one), and one or more **workspace researchers** that can access the data and workspace services in the workspace. The workspace owner is also considered a workspace researcher.
```

Comment:
@TessFerrandez you can assign more than one owner right? (It is a role assigned in AAD, right?)